Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Light's Vengeance

## Changelog:
- Added Light's Vengeance to Arthas' treasury in 2. bookmark

# How to test:
- Start the game in and bookmark, check Arthas' inventory
- _Should_ only appear in 2. bookmark "Reign of Chaos"
